### ISSUE TITLE
[dst/filterscripts] DSTExtractor setting SubEventID to 0 in L2 (Trac #1915)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1907]
Apparently DSTExtractor resets the event header, including the SubEventID.  So all the SubEventIDs in L2 are 0.

How it happens:
in the segment dst.extractor.ExtractDST:
   extract_to_frame = True
this causes the header to be overwritten.

Maybe we should set this to False in filterscripts? Not sure what else it controls though.

Migrated from https://code.icecube.wisc.edu/ticket/1915
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:30", 
    "description": "Apparently DSTExtractor resets the event header, including the SubEventID.  So all the SubEventIDs in L2 are 0.\n\nHow it happens:\nin the segment dst.extractor.ExtractDST:\n   extract_to_frame = True\nthis causes the header to be overwritten.\n\nMaybe we should set this to False in filterscripts? Not sure what else it controls though.", 
    "reporter": "david.schultz", 
    "cc": "claudio.kopper", 
    "resolution": "fixed", 
    "_ts": "1550067210114669", 
    "component": "combo reconstruction", 
    "summary": "[dst/filterscripts] DSTExtractor setting SubEventID to 0 in L2", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-11-29T22:22:46", 
    "milestone": "", 
    "owner": "juancarlos", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161213T20:09:24: 
* juancarlos changed resolution from "" to "fixed"
* juancarlos changed status from "new" to "closed"
* juancarlos commented:

> In changeset 152142:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="152142"
> closes #1907
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:30: 
* olivas commented:

> Milestone December 2016 deleted


* olivas changed milestone from "December 2016" to ""
